release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,861 @@
++
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: snapshot-promoter
+provisioner: volumesnapshot.external-storage.k8s.io/snapshot-promoter
+----
+
+. The tenant can create a snapshot of a PVC named `gce-pvc` and the resulting
+snapshot will be called `snapshot-demo`.
++
+----
+$ oc create -f snapshot.yaml
+
+apiVersion: volumesnapshot.external-storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot-demo
+  namespace: myns
+spec:
+  persistentVolumeClaimName: gce-pvc
+----
+
+. Now, they can restore their pod to that snapshot.
++
+----
+$ oc create -f restore.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: snapshot-pv-provisioning-demo
+  annotations:
+    snapshot.alpha.kubernetes.io/snapshot: snapshot-demo
+spec:
+  storageClassName: snapshot-promoter
+----
+
+[[ocp-311-scale]]
+=== Scale
+
+[[ocp-311-scale-cluster-limits]]
+==== Cluster Limits
+
+Updated guidance around
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[Cluster
+Limits] for {product-title} 3.11 is now available.
+
+[[ocp-311-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-311-prometheus]]
+==== Prometheus Cluster Monitoring
+
+Prometheus cluster monitoring is now fully supported in {product-title}.
+
+You can deploy Prometheus on an {product-title} cluster, collect Kubernetes and
+infrastructure metrics, and get alerts. You can see and query metrics and alerts
+on the Prometheus web dashboard. Alternatively, you can bring your own Grafana
+and hook it up to Prometheus.
+
+See xref:../install_config/cluster_metrics.adoc#openshift-prometheus[Prometheus
+on OpenShift] for more information.
+
+[[ocp-311-syslog-output-plugin-for-fluentd]]
+==== syslog Output Plug-in for fluentd (Technology Preview)
+
+syslog Output Plug-in for fluentd is a feature currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+You can send system and container logs from {product-title} nodes to external
+endpoints using the syslog protocol. The fluentd syslog output plug-in supports
+this.
+
+[IMPORTANT]
+====
+Logs sent via syslog are not encrypted and, therefore, insecure.
+====
+
+See
+xref:../install_config/aggregate_logging.adoc#sending-logs-to-external-rsyslog[Sending
+Logs to an External Syslog Server] for more information.
+
+[[ocp-311-developer-experience]]
+=== Developer Experience
+
+[[ocp-311-cli-plug-ins]]
+==== CLI Plug-ins (Technology Preview)
+
+CLI plug-ins are currently in xref:ocp-311-technology-preview[Technology Preview]
+and not for production workloads.
+
+Usually called _plug-ins_ or _binary extensions_, this feature allows you to
+extend the default set of `oc` commands available and, therefore, allows you to
+perform new tasks.
+
+See xref:../cli_reference/extend_cli.adoc#cli-reference-extend-cli[Extending the
+CLI] for information on how to install and write extensions for the CLI.
```

Comment:
CLI plug-ins feature was implemented since >= v3.7. It is by design to add it explicitly in v3.11 release notes?